### PR TITLE
AHU Canonical type

### PR DESCRIPTION
**Pull request by darshanjain12**

### CODE REVIEW

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 1]
```diff
@@ -1255,6 +1255,15 @@ CHPM:
   implements:
   - MONITORING
 
+EFSM:
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:1258]: nit: adjust to `ESFM`, so as not to confuse with `EF` for exhaust fan.

File: ontology/yaml/resources/HVAC/entity_types/AHU.yaml [hunk 2]
```diff
@@ -3600,6 +3600,21 @@ AHU_DFSS_DXZC_ECONZ_EFSS2X_HT2ZC_CO2C:
...
+  description: "Single zone AHU with CO2,VOC based outside air control,static pressure based VFD speed control,return air temperature based Chilled water valve modulation."
+  is_canonical: true
+  implements:
+  - AHU
+  - SFVSC
+  - CHWRC
+  - OAFC
+  - SSPC
+  - VOCC
+  - CO2C
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/AHU.yaml:3614]: these are zone level controls. which zone? 

i assume this is really return, or an average of zones (or max of zones). if so, need new VOCC and CO2C types to reflect that they are whatever they actually are.

**Review by tasodorff — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml (hunk 1 above)
Comment (darshanjain12) [reply ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:1258]: Sure. Done the required changes.

Comment (darshanjain12) [reply ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml]: Done the required changes.

File: ontology/yaml/resources/HVAC/entity_types/AHU.yaml [hunk 3]
```diff
@@ -3600,6 +3600,21 @@ AHU_DFSS_DXZC_ECONZ_EFSS2X_HT2ZC_CO2C:
   - EFSS2X
   - HT2ZC
   - CO2C
+  
+AHU_SFVSC_CHWRC_OAFC_SSPC_RAVOCC_RACO2C_ESFM:
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/AHU.yaml:3604]: You need to have some form of fan SS included, as well.

File: ontology/yaml/resources/HVAC/entity_types/AHU.yaml [hunk 4]
```diff
@@ -3600,6 +3600,21 @@ AHU_DFSS_DXZC_ECONZ_EFSS2X_HT2ZC_CO2C:
...
+  description: "Single zone AHU with CO2, VOC based outside air control, static pressure based VFD speed control,return air temperature based Chilled water valve modulation."
+  is_canonical: true
+  implements:
+  - AHU
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/AHU.yaml:3608]: AHU types need some form of `ECON` control, as well.

File: ontology/yaml/resources/HVAC/entity_types/AHU.yaml [hunk 5]
```diff
@@ -3600,6 +3600,21 @@ AHU_DFSS_DXZC_ECONZ_EFSS2X_HT2ZC_CO2C:
...
+  - SFVSC
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/AHU.yaml:3609]: If this is truly a single-zone unit, you want to use `discharge` instead of `supply`

Comment (darshanjain12) [reply ontology/yaml/resources/HVAC/entity_types/AHU.yaml:3609]: Yes . We can use supply when the AHU is supplying it to VAVs right?

File: ontology/yaml/resources/HVAC/entity_types/AHU.yaml (hunk 3 above)
Comment (darshanjain12) [reply ontology/yaml/resources/HVAC/entity_types/AHU.yaml]: Added

File: ontology/yaml/resources/HVAC/entity_types/AHU.yaml (hunk 4 above)
Comment (darshanjain12) [reply ontology/yaml/resources/HVAC/entity_types/AHU.yaml:3608]: Fresh air is controlled on basis of CO2 and VOC there is no economizer mode control  implemented.

Comment (cstirdivant) [reply ontology/yaml/resources/HVAC/entity_types/AHU.yaml:3608]: Ok

**Review by cstirdivant — CHANGES_REQUESTED** [review]

**Comment by mschulze17** [comment]
@tasodorff please review

File: ontology/yaml/resources/HVAC/entity_types/AHU.yaml [hunk 6]
```diff
@@ -3600,6 +3600,22 @@ AHU_DFSS_DXZC_ECONZ_EFSS2X_HT2ZC_CO2C:
...
+  - SFSS
+  - SFVSC
+  - CHWRC
+  - OAFC
+  - SSPC
+  - RAVOCC
+  - RACO2C
+  - ESFM
+  opt_uses:
+  - isolation_damper_status
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/AHU.yaml:3618]: which isolation damper? be specific, its not obvious which one it is based on this context.

**Review by tasodorff — CHANGES_REQUESTED** [review]

**Review by cstirdivant — APPROVED** [review]

**Pull request merged**